components/ChatApp.tsx

```diff
@@ -344,8 +308,8 @@ export class ChatApp extends React.Component<
                                                     .newChannel,
                                             },
                                         );
-                                        channel.add(this.username);
-                                        channel.add("coach");
+                                        channel.add(this.props.username);
+                                        channel.add("alex");
```

Comment:
I don't understand why you do it twice, because `channel.add("alex");` is the same as `channel.add(this.props.username);`

Reply:
Not really, alex is a coach, so when an employer create a channel, the coach have to be added too, but for sure I can conditionally do it to don't duplicate action when it's the coach who create a channel

Reply:
can you then write a comment to explain this